app/models/manageiq/providers/ibm_power_hmc/infra_manager/host.rb

```diff
@@ -1,6 +1,42 @@
 class ManageIQ::Providers::IbmPowerHmc::InfraManager::Host < ::Host
   include ManageIQ::Providers::IbmPowerHmc::InfraManager::MetricsCaptureMixin
 
+  supports :stop
+  supports :shutdown
+
+  def shutdown
+    $ibm_power_hmc_log.info("#{self.class}##{__method__}")
+    ext_management_system.with_provider_connection do |connection|
+      connection.poweroff_managed_system(ems_ref)
+    rescue IbmPowerHmc::Connection::HttpError => e
+      $ibm_power_hmc_log.error("error powering off LPAR #{ems_ref} with params=#{params}: #{e}")
```

Comment:
params is not defined here.
Also it is not an LPAR so the text should be:
```suggestion
      $ibm_power_hmc_log.error("error powering off managed system #{ems_ref}: #{e}")
```